packages/nft-analytics/src/controllers/twaps/input.ts

```diff
@@ -0,0 +1,30 @@
+import type { InputParameters } from '@chainlink/types'
+
+import { RequestMethod } from '../constants'
+
+export const sharedInputParameters: InputParameters = {
```

Comment:
The idea of `sharedInputParameters` is to be shared across all the DPs and endpoints.

A solution is having an `controllers/input.ts` and then keep the specific `input.ts` per DP (e.g. `controllers/floorprices/input.ts`, as you've already implemented).

Then let's check how each `inputParameters` behave when the documentation is generated